runtime/lua/vim/_options.lua

```diff
@@ -727,7 +727,13 @@ local function create_option_accessor(scope)
 
   return setmetatable({}, {
     __index = function(_, k)
-      return make_option(k, api.nvim_get_option_value(k, {}))
+      if scope == 'global' then
+        local global_value = api.nvim_get_option_value(k, { scope = 'global' })
+        return make_option(k, global_value)
+      else
+        local global_local_value = api.nvim_get_option_value(k, {})
+        return make_option(k, global_local_value)
+      end
```

Comment:
Since the only difference between the if and else is the options, the logic could be simplified
```suggestion
      local opts = {}
      if scope == 'global' then
        opts = { scope = 'global' }
      end
      
      return make_option(k, api.nvim_get_option_value(k, opts))
```

Reply:
Done. Thanks!